lang/cs/Org.Apache.REEF.Network/Group/Driver/Impl/TaskStarter.cs

```diff
@@ -32,7 +33,8 @@ namespace Org.Apache.REEF.Network.Group.Driver.Impl
     /// <summary>
     /// Helper class to start Group Communication tasks.
     /// </summary>
-    public class TaskStarter
+    [Private]
+    public sealed class TaskStarter
```

Comment:
TaskStarter now is used by client when they create drivers. It cannot be internal or private unless we provide other way for clients.